public/app/percona/dbaas/components/DBCluster/DBCluster.utils.tsx

```diff
@@ -78,24 +77,48 @@ export const getExpectedResourcesDifference = (
   };
 };
 
-export const formatDBClusterVersion = (version?: string) => (version ? version.split(':')[1].split('-')[0] : '');
+export const formatDBClusterVersion = (version?: string) => version;
 
 export const formatDBClusterVersionWithBuild = (version?: string) => (version ? version.split(':')[1] : '');
 
-const clustersToModel = (database: Databases, clusters: DBClusterPayload[], kubernetes: Kubernetes[], index: number) =>
-  clusters.map((cluster) => {
-    return newDBClusterService(database).toModel(cluster, kubernetes[index].kubernetesClusterName, database);
+const clustersToModel = (clusters: DBClusterListPayload[], kubernetes: Kubernetes[], index: number) =>
+  clusters.map((dbCluster) => {
+    let database = DATABASE_CLUSTER_TYPE[dbCluster.cluster_type] as Databases;
+    return {
+      id: dbCluster.id,
+      clusterName: dbCluster.name,
+      kubernetesClusterName: kubernetes[index].kubernetesClusterName,
+      databaseType: database,
+      installedImage: dbCluster.installed_image,
+      availableImage: dbCluster.available_image,
+    };
   });
 
 export const formatDBClusters = (results: any[], kubernetes: Kubernetes[]) => {
-  const clustersList: DBCluster[] = results.reduce((acc: DBCluster[], r, index) => {
-    const pxcClusters: DBClusterPayload[] = r.pxc_clusters ?? [];
-    const psmdbClusters: DBClusterPayload[] = r.psmdb_clusters ?? [];
-    const pxcClustersModel = clustersToModel(Databases.mysql, pxcClusters, kubernetes, index);
-    const psmdbClustersModel = clustersToModel(Databases.mongodb, psmdbClusters, kubernetes, index);
+  const clustersList: Array<Partial<DBCluster>> = results.reduce((acc: Array<Partial<DBCluster>>, r, index) => {
+    const dbClusters: DBClusterListPayload[] = r.db_clusters ?? [];
+    const dbClustersModel = clustersToModel(dbClusters, kubernetes, index);
 
-    return acc.concat([...pxcClustersModel, ...psmdbClustersModel]);
+    return acc.concat([...dbClustersModel]);
```

Comment:
Is there a need here to create a new array from `dbClusterModel`?

Reply:
no need, you are right, i'll fix it